iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/FileUploadTests.java

```diff
@@ -87,46 +86,7 @@
     protected static final String testProxyUser = "proxyUsername";
     protected static final char[] testProxyPass = "1234".toCharArray();
 
-    static Set<FileUploadNotification> activeFileUploadNotifications = new ConcurrentSkipListSet<>(new Comparator<FileUploadNotification>()
-    {
```

Comment:
Because our file upload notifications aren't comparable, the ConcurrentSkipListSet failed to work as a house for the received file upload notifications unless we provided a comparator in this test. The ConcurrentLinkedQueue did not require this, so I simplified our code by just using a ConcurrentLinkedQueue.